docs/content/cookbook/oss-deployment-guide.md

```diff
@@ -0,0 +1,625 @@
+       return 404; # managed by Certbot
+   
+   
+}
+```
+
+## Setting up Services
+### Overview
+`systemctl` is a command-line tool that allows for the management and monitoring of the `systemd` system and service manager. It is used to start, stop, and restart services, as well as to configure services to automatically start when the system boots. We'll use it to start our frontend and backend applications. To do this, we'll first create a service for each application.
+
+### Create a service for the frontend
+Create a file called vsfcommunity.service in `/etc/systemd/system/` and paste the following contents into it:
+```bash[/etc/systemd/system/vsfcommunity.service]
+[Unit]
+Description=vsfcommunity-web service
+Documentation=https://vsfcommunity.dev
+After=network.target
+
+
+[Service]
+Restart=always
+RestartSec=10
+TimeoutSec=300
+WorkingDirectory=/var/www/html/live/apps/web
+ExecStart=/usr/bin/bash -c 'yarn start'
+
+[Install]
+WantedBy=multi-user.target
+
+# /etc/systemd/system/vsfcommunity-web.service
+
+```
+
+### Create a service for the API
+Create a file called vsfcommunity-api.service in `/etc/systemd/system/` and paste the following contents into it:
+```bash[/etc/systemd/system/vsfcommunity-api.service]
+[Unit]
+Description=vsfcommunity-server service
+Documentation=https://vsfcommunity.dev
+After=network.target
+
+
+[Service]
+Restart=always
+RestartSec=10
+TimeoutSec=300
+WorkingDirectory=/var/www/html/live/apps/server
+ExecStart=/usr/bin/bash -c 'yarn start'
+
+[Install]
+WantedBy=multi-user.target
+
+# /etc/systemd/system/vsfcommunity-server.service
+```
+
+If you're new to systemd, here's a quick overview of how to use it:
+
+### To enable a service:
+```bash
+systemctl enable vsfcommunity-web
+```
+
+### To start a service:
+```bash
+systemctl start vsfcommunity-web
+```
+
+### To check the status of a service:
+```bash
+systemctl status vsfcommunity-web
+```
+Checking the status will give you information about the service, including whether it's running or not.
+![systemctl status image](./img/systemctl-status.png)
+
+### To stop a service:
+```bash
+systemctl stop vsfcommunity-web
+```
+:hr
+
+### Enable the services
+In a subsequent step, we will create a deployment script that will automatically start the services. However, we'll need to enable the services before they be run. To do this, run the following commands:
+
+```bash
+systemctl enable vsfcommunity-web
+```
+```bash
+systemctl enable vsfcommunity-api
+``` 
+
+#tab-2
+
+### Add SSH Key to Github Secrets
+We'll need to add an SSH key to our Github repository so that we can upload the tarball and run commands on our target server. Generate an SSH and make sure it's added to your server. Then, add the private key to your Github repository as a secret. To do this, go to your repository's settings page, click on "Secrets", and then click on "New repository secret". Name the secret "SSH_KEY" and paste the contents of your private key into the value field.
```

Comment:
```suggestion
We'll need to add an SSH key to our GitHub repository so that we can upload the tarball and run commands on our target server. Generate an SSH and make sure it's added to your server. Then, add the private key to your GitHub repository as a secret. To do this, go to your repository's settings page, click on "Secrets", and then click on "New repository secret". Name the secret "SSH_KEY" and paste the contents of your private key into the value field.
```

Capitalization, can you find and replace `Github` -> `GitHub`